src/ClientContextInterface.php

```diff
@@ -127,6 +127,7 @@
  * @method $this zrevrange($key, $start, $stop, array $options = null)
  * @method $this zrevrangebyscore($key, $max, $min, array $options = null)
  * @method $this zrevrank($key, $member)
+ * @method $this zunion(array $keys, int[] $weights = [], string $aggregate = 'sum', bool $withScores = false)
```

Comment:
Should keys and weights be a single array?

Reply:
Not sure If I get you. If you mean argument types, then I think both of them should be arrays.

For keys it's definitely should be, since we need to resolve numkeys argument and the easiest way is to calculate array count and we already have a trait for it. For weights it's optional, but in my opinion it's always easy to deal with single element the same way we're dealing with collection, in this case we don't need to maintain different implementations for different argument types